client-runtime/src/main/java/com/microsoft/rest/v2/policy/CredentialsPolicy.java

```diff
@@ -25,7 +25,7 @@
 
         /**
          * Creates a Factory which produces CredentialsPolicy.
-         * @param credentials The credentials to use for authentication.
+         * @param credentials The credentialsPolicyFactory to use for authentication.
```

Comment:
Bad IntelliJ. Bad!

Reply:
Gotta look at those refactor previews and diffs before just doing it..